.mvn/settings.xml

```diff
@@ -1,4 +1,8 @@
 <settings xmlns="http://maven.apache.org/SETTINGS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 http://maven.apache.org/xsd/settings-1.0.0.xsd">
+  <pluginGroups>
+    <pluginGroup>org.jenkins-ci.tools</pluginGroup>
+  </pluginGroups>
+
```

Comment:
Changes [required](https://www.jenkins.io/doc/developer/plugin-development/updating-parent/) in newer versions of Maven Parent POM.